File: C++/Patterns/number-pattern-2.cpp
// 1
// 2 3
// 4 5 6
// 7 8 9 10
// 11 12 13 14 15

#include<iostream>
using namespace std;

int main(){
    int i,j,r,num=1;
    cout<<"enter number of rows :";
    cin>>r;
    for(i=0;i<r;i++){
        for(j=0;j<=i;j++){
            cout<<" "<<num;
            num++;
        }
        cout<<endl;
    }
}

File: Python/Pattern/number-pattern-2.py
#  1
#  2 3
#  4 5 6
#  7 8 9 10
#  11 12 13 14 15
n=int(input("enter number of rows :"))
num=1;
for i in range(n):
    for j in range(i+1):
        print(num, end=" ")
        num+=1
    print()